.gitignore

```diff
@@ -20,4 +20,7 @@ testing/ef-tests/mainnet/*
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
-.idea/
\ No newline at end of file
+.idea/
+
+# redb files: prevent accidental commits of database files
+*.redb
```

Comment:
![image](https://github.com/user-attachments/assets/045cea53-d54d-4bb1-abbf-8d5e607998b3)

we are missing a new line